docs/root/api/api_supported_versions.rst

```diff
@@ -6,12 +6,11 @@ Supported API versions
 Envoy's APIs follow a :repo:`versioning scheme <api/API_VERSIONING.md>` in which Envoy supports
 multiple major API versions at any point in time. The following versions are currently supported:
 
-* :ref:`v2 xDS API <envoy_api_reference>` (*deprecated*, end-of-life EOY 2020). This API will not
-  accept new features after the end of Q1 2020.
 * :ref:`v3 xDS API <envoy_v3_api_reference>` (*active*, end-of-life unknown). Envoy developers and
```

Comment:
While you are in here, I noticed this recently and would suggest:

```suggestion
* :ref:`v3 xDS API <envoy_v3_api_reference>` (*active*). Envoy developers and
```

Reply:
Agreed. I'm pretty sure v4 is not going to happen anytime in the foreseeable future and I'm planning on removing tests/tooling as we go to that effect. OTOH, it will be worth doing the v2 -> v3 postmortem and making the rationale for this explicitly stated.